testdata/oracle/complex/complex-multi-table-delete.json

```diff
@@ -0,0 +1,16 @@
+{
+    "input": "DELETE FROM orders o WHERE o.customer_id IN (SELECT c.id FROM customers c WHERE NOT EXISTS (SELECT 1 FROM customer_orders co WHERE co.customer_id = c.id AND co.order_date > SYSDATE - 365)) AND EXISTS (SELECT 1 FROM order_items oi WHERE oi.order_id = o.id AND oi.product_id IN (SELECT p.id FROM products p WHERE p.category = 'Obsolete'));",
+    "outputs": [
+      {
+        "expected": "DELETE FROM orders o WHERE o.customer_id IN ( SELECT c.id FROM customers c WHERE NOT EXISTS ( SELECT ? FROM customer_orders co WHERE co.customer_id = c.id AND co.order_date > SYSDATE - ? ) ) AND EXISTS ( SELECT ? FROM order_items oi WHERE oi.order_id = o.id AND oi.product_id IN ( SELECT p.id FROM products p WHERE p.category = ? ) )",
```

Comment:
Do we want to add space before and after parentheses?

Reply:
It's controlled by normalizer option `RemoveSpaceBetweenParentheses`. By default it's disabled and spaces will be added. I do have add test cases to make sure this config option works as expected, for example https://github.com/DataDog/go-sqllexer/blob/04e119112021afe1fed98ecdb8fd1934011a933e/testdata/oracle/complex/complex-select-aggregates-joins.json#L18